engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetWireframe.java

```diff
@@ -23,7 +23,9 @@
  * TODO: Add javadocs
  */
 public final class SetWireframe implements StateChange {
-    private static SetWireframe defaultInstance = new SetWireframe(false);
+    private static final boolean DEFAULT_VALUE = false;
```

Comment:
I don't understand what was wrong with the simple approach for this class. The other state changes are more debatable because they need the FBM, but this class was good as it was.